OHM-Trade-Agent-v1/app/opip/data_platform/maintenance.py

```
@@ -109,14 +143,48 @@ def main(argv: Iterable[str] | None = None) -> int:
+            refresh_materialized_views(connection)
+            dropped = (
+                prune_analytical_retention(connection) if args.prune else []
+            )
+            record_maintenance_run(
+                connection,
+                status="SUCCESS",
+                detail=f"dropped_partitions={dropped}",
+                started_at=started_at,
+                finished_at=datetime.now(timezone.utc),
+            )
+    except Exception as error:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)